cookbook/composer.rst

```diff
@@ -14,35 +14,24 @@ following sections.
 Install Composer on Linux and Mac OS X
 --------------------------------------
 
-To install Composer on Linux or Mac OS X, execute the following two commands:
+#. Run the installer as described in `the official Composer documentation`_.
+#. Execute the following command to move the ``composer.phar`` to a directory that is in your path:
 
 .. code-block:: bash
 
-    $ curl -sS https://getcomposer.org/installer | php
     $ sudo mv composer.phar /usr/local/bin/composer
```

Comment:
please indent this block with 3 spaces (this way, it's part of the second item of the list)